surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/info/JUnit5ProviderInfo.java

```diff
@@ -11,28 +12,31 @@
 
     private String junit5SurefirePlatformVersion;
 
-    public JUnit5ProviderInfo() {
-        super(LoaderVersionExtractor.getJUnit5Version());
-        junit5SurefirePlatformVersion = retrieveJunit5SurefirePlatformVersion();
+    public JUnit5ProviderInfo(File projectDir) {
+        super(null);
```

Comment:
Any particular reason why extend from `JUnitProviderInfo` instead of implementing `ProviderInfo`? This `null` looks weird.

Reply:
good point - this is kind of zombie